### ISSUE TITLE
Exciting concept

### ISSUE THREAD

**Issue by S4-Hosting** [humzayunasnandla/CyberPanel-WordPress#1]
Hi, love the idea of WP and CP integration. Notice there is no readme file yet, could you just briefly explain the idea, scope and functions of the plugin?

**Comment by humzayunasnandla** [comment]
Thanks for taking interest in this project. Every billing platform is costly for hosting providers for automation. There are no design flexibility. So that's why i want to integrate all that to wordpress a great free platform where people can design there websites as they want they have freedom of everything. This plugin is just to connect to CyberPanel a free project already present on github. Users will be able to directly sell their hosting VIA automation from wordpress. This will save everyone costs of designs of messy billing platforms. Everyone will have a remarkable website using wordpress and they will sell hosting using this plugin. 

I hope you got an idea 
Regards,

**Comment by S4-Hosting** [comment]
Yep, I got it. 

We love CyberPanel and use it on a few servers and I'm really interested in checking out what kind of automation you are creating from a WP site. 

What's the installation procedure if we wanted to look at testing and contributing. I'm assuming it should be installed as any normal WP plugin?

**Comment by humzayunasnandla** [comment]
Yes you can install it as normal wp plugin. This is currently under development. Just the starting phase in completed that it can connected to cyberpanel api. That is done till now. I can also provide you some cheap server. Which provider currently you are using ? You can let me know i will provide and setup your servers with free cpanel or cyberpanel whatever you say and can setup reselller etc etc for you.
